security/tinc/src/opnsense/mvc/app/models/OPNsense/Tinc/Tinc.xml

```diff
@@ -22,7 +22,7 @@
                     <ValidationMessage>Please specify a valid hostname.</ValidationMessage>
                 </hostname>
                 <extaddress type="TextField">
-                    <Required>Y</Required>
```

Comment:
I think we could also use a `HostnameField` in combination with a tokeniser in the form, in which case you wouldn't need to explain the split character.